cylc/sphinx_ext/cylc_lang/__init__.py

```diff
@@ -214,5 +271,6 @@ def setup(app):
     app.add_domain(ParsecDomain)
     app.add_directive('auto-cylc-conf', CylcAutoDirective)
     app.add_directive('auto-cylc-type', CylcAutoTypeDirective)
+    app.add_directive('cylc-metadata', CylcMetadataDirective)
```

Comment:
I assume that we'll grow this into a full workflow documenter in due course so best to avoid using `metadata` in the name (restricts its scope).

Suggest `auto-cylc-workflow` and `auto-cylc-global` for directive names? Splitting the two also avoids having the orthogonal `:source:` and `:global:` options as we can just give the path, e.g:

```rest

.. automodule:: cylc.flow

.. autoclass:: cylc.flow.scheduler.Scheduler

.. auto-cylc-worklow:: ~/cylc-src/myworkflow

.. auto-cylc-global:: ~/.cylc/flow/global.cylc
```